src/components/SRNEntry.vue

```diff
@@ -100,35 +103,35 @@ export default {
   },
   data() {
     return {
-      userIDList: [{ userID: "", valid: false }], //array containing user-id's and a valid flag for each
+      userIDList: [{ userID: "", valid: false }], //array containing user-ids and a valid flag for each
       invalidInputMessage: null, // whether the input is in correct format
       isCurrentUserValid: false, // whether the current user is valid
       maxLengthOfSRN: 10,
-      validateCount: 0, //this variable tells us how many times the user has been validated.
+      validateCount: 0, // count the number of times the user has been validated
       invalidLoginMessage: "Please enter correct SRN / कृपया सही SRN दर्ज करें",
       isLoading: false,
     };
   },
   computed: {
     /** Returns length of the list of user IDs */
-    userIdListLength() {
+    numOfUserIds() {
       return this.userIDList.length;
     },
     /** Checks if any userID has been entered */
     isAnyUserIDPresent() {
       return (
         this.userIDList != undefined &&
-        this.userIdListLength > 0 &&
+        this.numOfUserIds > 0 &&
         this.userIDList[0]["userID"] != ""
       );
     },
     /** Whether multiple entries have been made by the user */
-    ifUserEnteredMoreThanOne() {
-      return !this.isSingleEntryOnly && this.userIdListLength > 1;
+    hasUserEnteredMoreThanOne() {
+      return !this.isMultipleIDEntryAllowedEntryOnly && this.numOfUserIds > 1;
```

Comment:
This name seems confusing to me i think! If anyone else also thinks so maybe can you think of an alternative?

Reply:
Sorry, that was a typo!